### ISSUE TITLE
Change color of selected elements in diagram

### ISSUE THREAD

**Issue by a19nikbo** [HGustavs/LenaSYS#9899]
Currently when you select an element the color of the element goes from pink to orange, but when taking color blindness into account it is easier to distinguish between tone/shade differences than color differences. Something we could try is to increase the default color brightness by about 10% and make the selection color the same color (pink) but 20% darker. Though keep in mind it still needs to be bright enough so that it is not difficult to read the black text.

**Comment by a19nikbo** [comment]
Fixed
